### ISSUE TITLE
inp_grab 0 causes cursor being stuck in top/left corner

### ISSUE THREAD

**Issue by TerminalMan** [teeworlds/teeworlds#1974]
TW Version 0.7.2
Environment: Archlinux + i3
SDL2 version: 2.0.9

inp_grab 0 causes the cursor to snap into the upper left corner after the first mouse input is registered. 0.6.X is also affected on my system.

edit: inp_grab 1 -> inp_grab 0. Dune is correct, i mixed it up.
edit2: added the SDL2 version

**Comment by heinrich5991** [comment]
Cannot reproduce on Arch with i3.

Does `inp_grab 0` work for you instead?

**Comment by Dune-jr** [comment]
@heinrich5991 We chatted on IRC beforehand and I believe this is wrongly reported. `inp_grab 1` fixes it, `inp_grab 0` causes this bug.

**Comment by oy** [comment]
@TerminalMan 
do you have an old SDL2 version?

**Comment by Dune-jr** [comment]
For the record:
> [quakenet] \<day> 2.0.9

**Comment by DevSnobo** [comment]
I think this is the problem I am having with teeworlds right now. Wanted to play again after 10 years, launched the game (also Arch + i3) and my mouse was wonky. Sliding way further than I want and not really able to move it to the center of my screen.

Could someone point me to where the wrong value is?

**Comment by Dune-jr** [comment]
@DevSnobo Try to type `inp_grab 1` in the console you get when you open `f1`. I think @TerminalMan is also a Arch + i3 nerd :-)

**Comment by DevSnobo** [comment]
Thanks for helping

Tried it, didn't help. It removed my cursor. Tried setting it back to 0 and the initial problem was back..

Edit (well, not really, but I tested something different):
It seems like Synergy was the problem. My setup is Arch as host, Win10 guest with gpu-passthrough for most gaming. For mouse/keyboard, I pass them to the VM and share them via Synergy.
Started the game, checked both options, both worked.

Anyways, thanks for sharing the F1 console part with me :)
